spec/requests/questions_spec.rb

```diff
@@ -0,0 +1,146 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Questions API', type: :request do
+  describe 'given we call index' do
+    subject { get api_v1_questions_path }
+
+    it 'should respond with successful HTTP status' do
+      subject
+
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'should respond with the current questions' do
+      foo_question = FactoryBot.create(:question);
+      bar_question = FactoryBot.create(:question);
+
+      subject
+
+      parsed_response = JSON.parse(response.body)
+      expect(parsed_response).to eq([foo_question, bar_question].as_json)
+    end
+  end
+
+  describe 'create' do
+    let(:question_params) do
+      { title: 'Nice Question is this what we looking for ?', order: 150, time_limit: 99, points: 40, answer_type: 'single', quiz_id: Quiz.first.id }
```

Comment:
There's no `Quiz.first` in this context! You either need to create a `Quiz` in a `before` statement, or better yet, use a `let(:quiz) { Quiz.create(...) }` and reference that!